.github/workflows/github_actions/actions/github_create_branch_action.rb

```diff
@@ -0,0 +1,13 @@
+class GithubCreateBranchAction
```

Comment:
I know there's not much plumbing there just yet, but should still make sure to make it a subclass of `Sublayer::Actions::Base` as we'll be adding logging and other things to the base class over time and would want these to all work similarly